clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -350,7 +352,8 @@ public void onGroupAssignmentUpdated(Set<TopicPartition> partitions) {
                     metrics,
                     fetchMetricsManager.throttleTimeSensor(),
                     clientTelemetryReporter.map(ClientTelemetryReporter::telemetrySender).orElse(null),
-                    backgroundEventHandler);
+                    backgroundEventHandler,
+                asyncConsumerMetrics);
```

Comment:
Nit: minor alignment issue:

```suggestion
                    backgroundEventHandler,
                    asyncConsumerMetrics);
```